File: PL2020/05/zadanie.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main() {

    ifstream file;
    file.open("input.txt");

    int number;
  
    for(int i = 0; i < 3; i++) {
        file >> number;
        cout << number << endl;
    }

    return 0;
}

File: 2020/9/zadanie.py
lastLetters = []

def isValid(number):
    for letter in lastLetters:
        if(number - letter) in lastLetters:
            return True
    return False

with open("input.txt") as f:
    for i in range(25):
        lastLetters.append(int(f.readline()))
    for line in f:
        newNumber = int(line)

        if(not isValid(newNumber)):
            print(line)
            break

        lastLetters.pop(0)
        lastLetters.append(newNumber)

